task/net/src/bsp/minibar.rs

```diff
@@ -0,0 +1,129 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#[cfg(not(all(feature = "ksz8463", feature = "mgmt", feature = "vlan")))]
+compile_error!("this BSP requires the ksz8463, mgmt, and vlan features");
+
+use crate::{
+    bsp_support::{self, Ksz8463},
+    mgmt, pins,
+};
+use drv_spi_api::SpiServer;
+use drv_stm32h7_eth as eth;
+use drv_stm32xx_sys_api::{Alternate, Port, Sys};
+use task_net_api::{
+    ManagementCounters, ManagementLinkStatus, MgmtError, PhyError,
+};
+use vsc7448_pac::types::PhyRegisterAddress;
+
+////////////////////////////////////////////////////////////////////////////////
+
+pub struct BspImpl(mgmt::Bsp);
+
+impl crate::bsp_support::Bsp for BspImpl {
+    // This system wants to be woken periodically to do logging
+    const WAKE_INTERVAL: Option<u64> = Some(500);
+
+    /// Stateless function to configure ethernet pins before the Bsp struct
+    /// is actually constructed
+    fn configure_ethernet_pins(sys: &Sys) {
+        pins::RmiiPins {
+            refclk: Port::A.pin(1),
+            crs_dv: Port::A.pin(7),
+
+            // The three pins on port G are incorrect as-built and must be
+            // reworked!
+            tx_en: Port::G.pin(11),
+            txd0: Port::G.pin(13),
+            txd1: Port::G.pin(12),
+
+            rxd0: Port::C.pin(4),
+            rxd1: Port::C.pin(5),
+            af: Alternate::AF11,
+        }
+        .configure(sys);
+
+        pins::MdioPins {
+            mdio: Port::A.pin(2),
+            mdc: Port::C.pin(1),
+            af: Alternate::AF11,
+        }
+        .configure(sys);
+    }
+
+    fn preinit() {
+        // Nothing to do here, XXX should we sleep?
```

Comment:
do we want to sleep?

Reply:
Seems to work fine without it, so I'll delete that XXX